locales/en/template.json

```diff
@@ -1,60 +1,78 @@
 {
+  "action_icons": {
+      "delete": "Delete",
+      "edit": "Edit"
+  },
   "add_field": "Add Field",
   "create": {
-    "title": "Create New Template"
+      "title": "Create New Template"
```

Comment:
have you done yarn lint ?

Reply:
I have done that and it shows No EsLint warnings or errors